packages/flutter_localizations/lib/src/l10n/material_en_AU.arb

```diff
@@ -8,30 +8,21 @@
   "keyboardKeyNumpad6": "Num 6",
   "keyboardKeyNumpad7": "Num 7",
   "keyboardKeyNumpad2": "Num 2",
-  "keyboardKeyKanaMode": "Kana mode",
```

Comment:
For my education, why are we removing these `keyboardKeyxxx: xxx`

Reply:
Because they were removed from the main list of localizations, but needed to also be removed from the .arb files: basically just left over cruft from having been removed elsewhere.

Reply:
Ah got it! Thanks👍